src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.Composite.sfxproj

```diff
@@ -82,9 +82,10 @@
 
     <NativeRuntimeAsset Condition=" '$(UseIisNativeAssets)' == 'true' AND !$(BuildNative) "
         Include="$(ArtifactsBinDir)InProcessRequestHandler\$(NativePlatform)\$(Configuration)\aspnetcorev2_inprocess.dll" />
+    <NativeRuntimeAsset Condition=" '$(UseIisNativeAssets)' == 'true' AND !$(BuildNative) "
```

Comment:
Usually other repos have done this in `_SymbolFilesToPackage`, but I'll let Jeremy comment on the more idiomatic way of doing this

Reply:
Yeah I prefer `NativeRuntimeAsset` over `_SymbolFilesToPackage`.